heracal/omni.py

```diff
@@ -219,8 +219,39 @@ def compute_reds(nant, pols, *args, **kwargs):
             reds += [[(Antpol(i, pi, nant), Antpol(j, pj, nant)) for i, j in gp] for gp in _reds]
     return reds
 
-
-def aa_to_info(aa, pols=['x'], fcal=False, **kwargs):
+def reds_for_minimal_V(reds):
+    '''
+    Manipulate redundancy array to combine crosspols
+    into a single redundancy array - imposing that
+    Stokes V = 0.
+
+    This works in the simple way that it does because
+    of the way the reds arrays are constructed in
+    aa_to_info when 4 polarizations are present: it
+    reprsents them as 4 co-located arrays in (NS,EW) and
+    displaced in z, with the cross-combinations (e.g. 
+    polarization xy and yx) _always_ in the middle.
+    
+    Args:
+        reds: the array of redundant baseline sets.
+        
+    Return:
+        _reds: the adjusted array of redundant baseline sets.
+    '''
+    _reds = []
+    n = len(reds)
+    if n%4 != 0:
+        raise ValueError('Wrong number of polarizations supplied')
```

Comment:
If you only have `pols=['x']` in `omni.compute_reds`, `len(reds)=2`, so the ValueError would be raised appropriately, since you can't have cross-pols in that array.

If you have  `pols=['x','y']`,  `len(reds)=8`, so the method will proceed, since cross-pols can be created. 

Even numbers of polarizations will not raise a ValueError. So you could imagine calculating the redundancy array for a future configuration where we have every-other feed rotated 45 degress as in the polarization experiment array, and it would construct an array that minimized the difference of the cross products of those polarizations.... whether or not they should be permitted to do so. But let's cross that bridge when we come to it!